tests/integration/records/table_timezone_validator.py

```diff
@@ -135,7 +135,7 @@ def validate(self,
 
         if self.tc.raw_avro_types_written():
             assert timestamptzstr == '946834496789012'
-        elif (self.source_db_engine is not None and
+        elif (self.source_db_engine is None or
```

Comment:
why did this change from `not None` to `None`? that looks wrong

Reply:
After the switch to SQLAlchemy 2.0, when we have a `None` db_engine, the output is now:
`2000-01-02 17:34:16.120786 ` (for example) whereas it used to be `2000-01-02 17:34:16.120786+00`. I was looking into this but I don't have a good reason why this changed.